dlt/sources/helpers/rest_client/paginators.py

```diff
@@ -64,7 +64,133 @@ def update_request(self, request: Request) -> None:
         return
 
 
-class OffsetPaginator(BasePaginator):
+class BaseNumericPaginator(BasePaginator):
```

Comment:
@rudolfix, not sure if the name should be "Base*", this class could be useful on its own. What do you think?

Reply:
Maybe `Simple*` is better for tbh both look fine